website/docs/r/subscription.html.markdown

```diff
@@ -0,0 +1,94 @@
+---
+subcategory: "Base"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_subscription"
+description: |-
+  Manages a Subscription.
+---
+
+# azurerm_subscription
+
+Manages a Subscription.
+
+~> **NOTE:** Destroying a Subscription controlled by this resource will place the Subscription into a cancelled state. It is possible to re-activate a subscription within 90-days of cancellation, after which time the Subscription is irrevocably deleted, and the Subscription ID cannot be re-used. For further information see [here](https://docs.microsoft.com/en-us/azure/cost-management-billing/manage/cancel-azure-subscription#what-happens-after-subscription-cancellation). Users can optionally delete a Subscription once 72 hours have passed, however, this functionality is not suitable for Terraform. A `Deleted` subscription cannot be reactivated.
+
+~> **NOTE:** It is not possible to destroy (cancel) a subscription if it contains resources. If resources are present that are not managed by Terraform then these will need to be removed before the Subscription can be destroyed.
```

Comment:
should we also add a note here about assuming a 1:1 between an alias and a subscription